debian/rules

```diff
@@ -169,9 +169,35 @@ install: build
 	for i in `find * -path 'docs/images/*' -type f`; do \
 		install -m0644 $$i $(CURDIR)/debian/packetfence-doc$(PREFIX)/$(NAME)/html/pfappserver/root/static/images/; \
 	done
+	#Portal javascript/css
+	make -C html/common/ vendor
+	make -C html/common light-dist
+	install -d -m0755 $(CURDIR)/debian/packetfence-captive-portal-javascript$(PREFIX)/$(NAME)/html/common
+	install -m0644 $(CURDIR)/html/common/styles.css $(CURDIR)/debian/packetfence-captive-portal-javascript$(PREFIX)/$(NAME)/html/common/
+	install -m0644 $(CURDIR)/html/common/styles.css.map $(CURDIR)/debian/packetfence-captive-portal-javascript$(PREFIX)/$(NAME)/html/common/
+	# Admin javascript/css
+	rm -f $(CURDIR)/debian/packetfence$(PREFIX)/$(NAME)/html/common/styles.css
+	rm -f $(CURDIR)/debian/packetfence$(PREFIX)/$(NAME)/html/common/styles.css.map
+	make -C html/pfappserver/root/static/ vendor
```

Comment:
We also need grunt (`sudo npm install -g grunt-cli`) and bower (`sudo npm install -g bower`). Should we install them from the Makefile?

Reply:
I did it in the chroot directly but i also changed in the Makefile